tests/functional/csp-simple.js

```diff
@@ -0,0 +1,13 @@
+require.config({
```

Comment:
Mainly just because I brought them over from some code written in 2015 :/ Like you said though, they are teeny, so i can convert them over!